rest/functions_api_test.go

```diff
@@ -92,13 +93,12 @@ func testConcurrently(t *testing.T, rt *RestTester, testFunc func() bool) bool {
 }
 
 func TestFunctions(t *testing.T) {
-	rt := NewRestTester(t, &RestTesterConfig{GuestEnabled: true, EnableUserQueries: true})
+	rt := NewRestTester(t, &RestTesterConfig{GuestEnabled: true, EnableUserQueries: true, DatabaseConfig: kGraphQLTestConfig})
 	defer rt.Close()
-	rt.DatabaseConfig = kGraphQLTestConfig
 
 	t.Run("GraphQL with variables", func(t *testing.T) {
 		testConcurrently(t, rt, func() bool {
-			response := rt.SendRequest("POST", "/db/_graphql",
+			response := rt.SendRequest("POST", fmt.Sprintf("/%s/_graphql", rt.GetDatabase().Name),
```

Comment:
Can this be `{{.db}}` (and other usages in functions_api_test.go)?

Reply:
done.